conf/default.env

```diff
@@ -0,0 +1,58 @@
+APP_NAME="Invoice Ninja"
+APP_ENV=production
+APP_KEY=base64:3E5HVZ/T+VbZu+oA4GNlwnvwFe3jNtkGbR95K0uwr7A=
+APP_DEBUG=false
+
+APP_URL=https://__DOMAIN____PATH__
+
+DB_CONNECTION=mysql
+MULTI_DB_ENABLED=false
+
+DB_HOST=localhost
+DB_DATABASE=__DB_NAME__
+DB_USERNAME=__DB_NAME__
+DB_PASSWORD=__DB_PWD__
+DB_PORT=3306
+
+DEMO_MODE=false
+
+BROADCAST_DRIVER=log
+LOG_CHANNEL=stack
+CACHE_DRIVER=file
+SESSION_DRIVER=file
+SESSION_LIFETIME=120
+
+REDIS_HOST=127.0.0.1
+REDIS_PASSWORD=null
+REDIS_PORT=6379
+
+MAIL_MAILER=smtp
+MAIL_HOST=smtp.__DOMAIN__
+MAIL_PORT=25
+MAIL_USERNAME=null
+MAIL_PASSWORD=null
+MAIL_ENCRYPTION=tls
+MAIL_FROM_ADDRESS='__EMAIL__'
+MAIL_FROM_NAME="__EMAIL_FIRSTNAME__ __EMAIL_LASTNAME__"
+
+POSTMARK_API_TOKEN=
+REQUIRE_HTTPS=false
+
+GOOGLE_MAPS_API_KEY=
+ERROR_EMAIL=
+TRUSTED_PROXIES=
+
+NINJA_ENVIRONMENT=selfhost
+
+#options - snappdf / phantom / hosted_ninja
+PDF_GENERATOR=phantom 
+
+PHANTOMJS_KEY='__PHANTOMJS_KEY__'
+PHANTOMJS_SECRET=__APP_KEY__
+
+UPDATE_SECRET=__API_SECRET__
+
+COMPOSER_AUTH='{"github-oauth": {"github.com": "${{ secrets.GITHUB_TOKEN }}"}}'
+SENTRY_LARAVEL_DSN=https://cc7e8e2c678041689e53e409b7dba236@sentry.invoicing.co/5
```

Comment:
This looks weird too ... is it okay to have this token here ?

Reply:
You're right, I copy-pasted this from somewhere. Will remove it.